ikarus/controlroutines/pathfollowing.inl

```diff
@@ -23,51 +23,54 @@ namespace Ikarus {
 
 template <typename NLS, typename PF, typename ASS>
 requires(Impl::checkPathFollowingTemplates<NLS, PF, ASS>())
-ControlInformation PathFollowing<NLS, PF, ASS>::run() {
-  ControlInformation info;
+ControlState PathFollowing<NLS, PF, ASS>::run() {
+  ControlState controlState{
+      .success = false, .currentStep = 0, .totalIterations = 0, .stepSize = stepSize_, .name = this->name()};
   auto& nonOp = nonLinearSolver_->nonLinearOperator();
-  this->notify(ControlMessages::CONTROL_STARTED, pathFollowingType_.name());
+  this->notify(ControlMessages::CONTROL_STARTED, controlState);
 
   SubsidiaryArgs subsidiaryArgs;
 
-  info.totalIterations    = 0;
   subsidiaryArgs.stepSize = stepSize_;
   subsidiaryArgs.DD.resizeLike(nonOp.firstParameter());
   subsidiaryArgs.DD.setZero();
+  subsidiaryArgs.Dlambda = 0.0;
 
   /// Initializing solver
-  this->notify(ControlMessages::STEP_STARTED, 0, subsidiaryArgs.stepSize);
-  pathFollowingType_.initialPrediction(nonOp, subsidiaryArgs);
-  auto solverInfo = nonLinearSolver_->solve(pathFollowingType_, subsidiaryArgs);
-  info.solverInfos.push_back(solverInfo);
-  info.totalIterations += solverInfo.iterations;
-  if (not solverInfo.success)
-    return info;
-  this->notify(ControlMessages::SOLUTION_CHANGED);
-  this->notify(ControlMessages::STEP_ENDED);
-
-  /// Calculate predictor for a particular step
-  for (int ls = 1; ls < steps_; ++ls) {
+  /// Dummy execution of the code inorder to save information of the undeformed (or initial)
```

Comment:
These parts of the code are modified only because this PR also unifies the output of `LoadControl` and `PathFollowing`, which affects the interface the observer patterns and hence it is not split into two PRs